operations/customers.md

```diff
@@ -569,54 +659,250 @@ Updates personal information of a customer. Note that if any of the fields is le
+  "Client": "Sample Client 1.0.0",
+  "Name": "Smith",
+  "ResourceId": null,
+  "Extent": {
+    "Customers": true,
+    "Documents": false,
+    "Addresses": false
+  }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Name` | string | optional | Name to search by (applies to first name, last name, and full name). |
+| `ResourceId` | string | optional | Identifier of [Resource](resources.md#resource) to search by (members of reservation assigned there will be returned). |
+| `Extent` | [Customer extent](customers.md#customer-extent) | optional | Extent of data to be returned. |
+
+### Response
+
+```javascript
+{
+  "Customers": [
+    {
+      "Customer": {
+        "Address": null,
+        "BirthDate": null,
+        "BirthPlace": null,
+        "CategoryId": null,
+        "Classifications": [],
+        "CreatedUtc": "2016-01-01T00:00:00Z",
+        "Email": null,
+        "FirstName": "John",
+        "Sex": "Male",
+        "Id": "35d4b117-4e60-44a3-9580-c582117eff98",
+        "IdentityCard": null,
+        "LanguageCode": null,
+        "LastName": "Smith",
+        "LoyaltyCode": null,
+        "NationalityCode": "US",
+        "Notes": "",
+        "Number": "1",
+        "Options": [],
+        "Passport": null,
+        "Phone": "00420123456789",
+        "SecondLastName": null,
+        "TaxIdentificationNumber": null,
+        "Title": null,
+        "UpdatedUtc": "2016-01-01T00:00:00Z",
+        "Visa": null
+      },
+      "Reservation": null
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Customers` | array of [Customer search result](customers.md#customer-search-result) | required | The customer search results. |
+| `Documents` | array of [Identity document](customers.md#identity-document) | optional | The identity documents of customers. |
+
+#### Customer search result
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Customer` | [Customer](customers.md#customer) | required | The found customer. |
+| `Reservation` | [Reservation (ver 2017-04-12)](reservations.md#reservation-ver-2017-04-12) | optional | Reservation of the customer in case they are currently staying in the property. |
 
 ## ~~Merge customers~~
 
 > ### Deprecated!
-> This operation is [deprecated](../deprecations/README.md), please use [Merge accounts](accounts.md#merge-accounts) instead.
-
-Merges one customer to another. All entities attached to the source customer \(e.g. orders, bills\) are attached to the target customer. Profile information of the target customer are extended but not overwritten with profile information of the source customer.
```

Comment:
Should we delete description when it's being deprecated? 🤔

Reply:
Nah, let's keep it for now, we can later sweep it.